schema.sql

```diff
@@ -58,4 +58,12 @@ CREATE TABLE visits (
   date DATE,
   CONSTRAINT fk_animals FOREIGN KEY(animals_id) REFERENCES animals(id),
   CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animals_id_index ON visits(animals_id);
+
+CREATE INDEX vets_id_index ON visits(vets_id, animals_id, date);
+
+CREATE INDEX email_index ON owners(email ASC);
```

Comment:
## Optional
- [ ] As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));` you can consider that

Reply:
Thank you for the suggestion, @DeliceLydia